config/config.go

```diff
@@ -881,30 +881,30 @@ type TxIndexConfig struct {
 	//   2) "kv" (default) - the simplest possible indexer, backed by key-value storage (defaults to levelDB; see DBBackend).
 	Indexer string `mapstructure:"indexer"`
 
-	// Comma-separated list of tags to index (by default the only tag is "tx.hash")
+	// Comma-separated list of events to index (by default the only tag is "tx.hash")
 	//
 	// You can also index transactions by height by adding "tx.height" tag here.
 	//
-	// It's recommended to index only a subset of tags due to possible memory
+	// It's recommended to index only a subset of events due to possible memory
 	// bloat. This is, of course, depends on the indexer's DB and the volume of
 	// transactions.
-	IndexTags string `mapstructure:"index_tags"`
+	IndexEvents string `mapstructure:"index_events"`
```

Comment:
problem here events is more complex data structure, so we'll need to explain how events can be transformed into `Comma-separated list of events` string

Reply:
also, not clear what an event is in this context? just `Type` or `Type.TagName`

Reply:
https://github.com/tendermint/tendermint/blob/master/abci/types/types.proto#L245 i believe tag should be replaced throughout the repo to event